caesar.c

```diff
@@ -6,29 +6,36 @@
 #include <stdlib.h>
 #include <err.h>
 
-void cipher(char *string, int shiftkey){
-    int  i;
-    for (i=0; i < strlen(string); i++){
```

Comment:
`strlen(string)` will loop over the string every iteration. Maybe, the compiler can optimise this, but... I doubt it.